appinfo/info.xml

```diff
@@ -2,12 +2,12 @@
 <info xmlns:xsi= "http://www.w3.org/2001/XMLSchema-instance"
       xsi:noNamespaceSchemaLocation="https://apps.nextcloud.com/schema/apps/info.xsd">
     <id>monthly_status_email</id>
-    <name>Monthly Status Email</name>
-    <summary>Monthly notifications with storage usage and tips and tricks</summary>
-    <description>Monthly notifications with some storage usage information and tips and tricks.</description>
+    <name>Monthly Status Mails</name>
+    <summary>Send monthly status mails to users</summary>
+    <description>Monthly notifications, e.g. summary of used storage and usage hints.</description>
     <version>1.0.9</version>
     <licence>agpl</licence>
-    <author>Carl Schwan</author>
+    <author>Jonas Meurer</author>
```

Comment:
```suggestion
    <author>Carl Schwan</author>
    <author>Jonas Meurer</author>
```

Reply:
Carl Schwan is no longer employed with us, so he will no longer maintain the app.

Reply:
But is the author.
I think that author never change, only is added new authors, not?!

Search at server repository by `<author>` in all files `info.xml`

Isn't about maintainers I think. Maybe would be a good question to ask to other engineers. If is about maintainers, we will need to change this by example:

https://github.com/nextcloud/server/blob/master/apps/dav/appinfo/info.xml#L10

Reply:
Well this field is called `author` not `maintainer` ;)

Reply:
Ah, good to know. Changed now.

Reply:
Sorry @CarlSchwan, I didn't want to rewrite history. I simply was wrong here :wink: